### PR TITLE
reach elements are created

### PR DESCRIPTION
**Pull request by harishkumar-srijan**
reach elements are created

### CODE REVIEW

File: stories/Atom/Reach-element/Table/table.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+The <code>table</code> HTML element represents tabular data — that is, the information presented in a two-dimensional table comprised of rows and columns of cells containing data. it's used to make large volumes of data easy to understand and access.
+
+<ul>
+<li>Table component created by <code>table</code> <code>tr</code> <code>th</code> and <code>td</code> tag.</li>
+</ul>
+
+<Story name="Table">
+  {(args, { globals: { locale } }) => {
+    const caption = getCaptionForLocale(locale);
+    //return <Breadcrumb blockquote={caption.blockquote} cite={caption.cite}></Breadcrumb>;
```
Comment (surbhi90) [inline stories/Atom/Reach-element/Table/table.stories.mdx:43]: @harishkumar-srijan remove commented code

File: stories/assets/scss/_breakpoints.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+            @content;
+        }
+    }
+    @else if $point==large {
+        /* large design */
+        @media (min-width: 1441px) {
+            @content;
+        }
+    }
+}
```
Comment (surbhi90) [inline stories/assets/scss/_breakpoints.scss:20]: @harishkumar-srijan these should be the breakpoints
/* Mobiles only */
@media screen and (max-width: 47.9375em) {}

/* Above Mobiles */
@media screen and (min-width: 48em) {}

/* Tablets only */
@media screen and (min-width: 48em) and (max-width: 89.9375em) {}

/* Large Desktops */
@media screen and (min-width: 90em) {}

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the mentioned commentd

File: stories/Atom/Reach-element/Details/details.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+The details tag creates a disclosure widget in which information is visible only when the widget is toggled into an "open" state. A summary or label must be provided using the summary element.
+
+A disclosure widget is typically presented onscreen using a small triangle that rotates (or twists) to indicate open/closed status, with a label next to the triangle. The contents of the summary element are used as the label for the disclosure widget.
+
+<ul>
+<li>Details component created by detail tag and summary tag.</li>
+<li>Custom arrow Chevron-small-down icon used in the open and close statement..</li>
+</ul>
+
```
[the remark this answers is not in the document]
Comment (srijan-mehul-gada) [reply stories/Atom/Reach-element/Details/details.stories.mdx:39]: I agree with Surbhi

File: stories/assets/scss/_variables.scss [hunk 4]
```diff
@@ -1,2 +1,49 @@
+// image variables
 $img-path: '../../../assets/images/';
-$img-path-Icon: '../../../assets/images/Icon/'
+$img-path-Icon: '../../../assets/images/Icon/';
+
+/* variables start */
+@import url('https://fonts.googleapis.com/css2?family=Lato&family=Nunito+Sans&display=swap');
+// COLORS
+$black: #000;
```
Comment (surbhi90) [inline stories/assets/scss/_variables.scss:8]: @harishkumar-srijan Please update the color variables from color

**Review by surbhi90 — COMMENTED** [review]
Please check comments

**Review by surbhi90 — APPROVED** [review]
LGTM

**Pull request merged**